lib/middleware/assignActiveLicence.js

```diff
@@ -0,0 +1,44 @@
+const { acquisitionCtxClient } = require('@financial-times/kat-client-proxies');
+
+const RE_FTCOM = /^FT\.com\s*/i;
+
+function addLicenceAcquisitionData (licence) {
+	return acquisitionCtxClient.getContexts({'access-licence-id': licence.licenceId})
+		.then(result => {
```

Comment:
Does this need a `.catch(next)`?

Reply:
Don't think so... having taken a second glance not sure why addLicenceAcquisitionData needs to be made available. I think it should be a private function for this module. I need to check where this is being used outside of being part of this middleware.